rule.proto

```diff
@@ -0,0 +1,221 @@
+syntax = "proto2";
+package dv360_excel_plugin;
+
+/* A message that represents a scheduled rule for manipulating DV360 entities.
+ */
+message Rule {
+  /* required
+   * The name of the rule, set by the user.
+   */
+  optional string name = 1;
+
+  /* required
```

Comment:
this is required for get but should be generated by server when creating, correct?

Reply:
Yes, I should remove the required comment :)